### PR TITLE
Add Hill Giant & Obor

### PR DESCRIPTION
**Pull request by jamosaur**
### Description:

- Added hill giants and obor
- relates to https://github.com/oldschoolgg/oldschoolbot/issues/377

### Changes:

-   [] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/simulation/monsters/low/HillGiant.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	.every('Big Bones')
+	.tertiary(25, 'Ensouled giant head')
+	.tertiary(50, 'Clue scroll (beginner)')
+	.tertiary(400, 'Long bone')
+	.tertiary(5000, 'Giant champion scroll')
+	.tertiary(5013, 'Curved bone')
+
+	/* Weapons and armour */
+	.add('Iron full helm', 1, 5)
+	.add('Iron dagger', 1, 5)
```
Comment (Alexsuperfly) [inline src/simulation/monsters/low/HillGiant.ts:16]:
```suggestion
	.add('Iron dagger', 1, 4)
```

File: src/simulation/monsters/low/HillGiant.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	.add('Chaos rune', 2, 1)
+	.add('Death rune', 2, 1)
+
+	/* Coins */
+
+	.add('Coins', 38, 14)
+	.add('Coins', 52, 10)
+	.add('Coins', 15, 8)
+	.add('Coins', 8, 6)
+	.add('Coins', 64, 2)
```
Comment (Alexsuperfly) [inline src/simulation/monsters/low/HillGiant.ts:38]:
```suggestion
	.add('Coins', 88, 2)
```

File: src/simulation/monsters/low/HillGiant.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+	/* Other */
+
```
Comment (Alexsuperfly) [inline src/simulation/monsters/low/HillGiant.ts:41]: you forgot the gemDropTable at a weight of 3

File: src/simulation/monsters/bosses/Obor.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	.add('Coins', [10_000, 20_000], 10)
+	.add('Limpwurt root', 20, 8)
+	.add('Big bones', 50, 8)
+	.add(
+		[
+			['Uncut Diamond', 5],
+			['Uncut ruby', 5]
+		],
+		1,
+		1
```
Comment (Alexsuperfly) [inline src/simulation/monsters/bosses/Obor.ts:42]:
```suggestion
		5
```

Comment (Alexsuperfly) [reply src/simulation/monsters/bosses/Obor.ts:42]: you did this backwards, it goes items, quant, weight
you want to drop this item array 1 time at 5 weight, not 5 times at 1 weight

**Comment by jamosaur** [comment]
> LGTM, only tests failing are api requests to the wiki which seem unrelated

Yeah, that was to do with the cloudflare outage last night 🙂

**Pull request merged**